.github/workflows/pre-merge.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: pre-merge
+
+on:
+  push:
+    branches:
+      - 'user/**'
+      - 'feature/**'
+      - 'improvement/**'
+      - 'bugfix/**'
+      - 'w/**'
+      - 'q/**'
+
+
+jobs:
+  test-states-syntax:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Use this action to freez the node version
https://github.com/actions/setup-node

Reply:
```suggestion
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '10'
          cache: 'npm'
```